src/libtomahawk/database/DatabaseCommand_ImportPlaylist.h

```diff
@@ -22,10 +22,18 @@
 #include <QObject>
 #include <QVariantMap>
 
+#include "DatabaseImpl.h"
+#include "DatabaseCommandLoggable.h"
+#include "Source.h"
+#include "Typedefs.h"
```

Comment:
Not sure about Typedefs, but none of the includes above is needed in this file.
DatabaseImpl can be forward declared, the other types don't occur in this file.

Please add them where needed, possibly the .cpp file or some other header using this one.

Reply:
typedefs is needed in this file, without it compilation fails in clang. added the others just in case, but can test without them

Reply:
What version of clang? What's the error?